scripts/starphleet-nginx-acl-rules

```
@@ -0,0 +1,61 @@
+#!/usr/bin/env starphleet-launcher
+### Usage:
+###    starphleet-nginx-acl-rules
+### --help
+###
+### Generate all ACL Rules for NGINX
+###
+
+die_on_error
+run_as_root_or_die
+
+#******************************************************************
+# Configuration
+#******************************************************************
+
+NGINX_CONF=${NGINX_CONF:-/var/starphleet/nginx}
+DIR_NGINX_ACL_RULES="${NGINX_CONF}/acl_rules"
+DIR_NGINX_PUBLISHED="${NGINX_CONF}/published"
+
+#******************************************************************
+# Main
+#******************************************************************
+
+# We purge everything each run and start over
+[ -d "${DIR_NGINX_ACL_RULES}" ]  && rm -rf "${DIR_NGINX_ACL_RULES}"
+rm "${DIR_NGINX_PUBLISHED}"/*.acl 2> /dev/null || true
```

Do not want to hide errors, if there's something unexpected.... we'll never be able to see it and will have to remove the 2> /dev/null before we can figure out what it is.